Source/Core/Core/AchievementManager.cpp

```diff
@@ -1239,7 +1252,7 @@ void AchievementManager::HandleGameCompletedEvent(const rc_client_event_t* clien
     return;
   }
   bool hardcore = rc_client_get_hardcore_enabled(client);
-  OSD::AddMessage(fmt::format("Congratulations! {} has {} {}", user_info->display_name,
+  OSD::AddMessage(fmt::format("Congratulations {}! You have {} {}", user_info->display_name,
```

Comment:
```suggestion
  OSD::AddMessage(fmt::format("Congratulations, {}! You have {} {}.", user_info->display_name,
```
The period may be fine to leave off, though. It looks like the majority of achievement-related messages do so, while the majority of other messages that are complete sentences include it.